icons/README.md

```diff
@@ -74,6 +74,10 @@ There is 2 types of icons
 #### Exposed Pod with 3 replicas
 ![](./docs/k8s-exposed-pod.png)
 
+### Slide Deck
+
+[Kubernetes_Icons_GSlide](https://docs.google.com/presentation/d/15h_MHjR2fzXIiGZniUdHok_FP07u1L8MAX5cN1r0j4U/edit)
```

Comment:
why do we need this?

Reply:
This kind of sample was initially included as it a much easier way to start with a sample slide deck instead on importing all PNG files.

After some color change, we had to update it.

Reply:
@yastij

Reply:
@cblecker 

+100 this is super useful to start with, I've been using it extensively as starter for my slide-deck.